### PR TITLE
Log channel send error in fullnode

### PR DESCRIPTION
**Pull request by rtso**

### CODE REVIEW

File: ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs [hunk 1]
```diff
@@ -187,9 +187,21 @@ impl IndexerStreamCoordinator {
...
+                    // Error from closed channel. This means the client has disconnected.
+                    error!(
+                        start_version = first_version,
+                        end_version = end_version,
+                        num_transaction,
+                        highest_known_version,
+                        duration_in_secs = sending_start_time.elapsed().as_secs_f64(),
+                        error = format!("{:?}", e),
+                        "[Indexer Fullnode] Error sending transactions to client."
+                    );
```
Comment (bowenyang007) [inline ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs:202]: just log no metrics?

File: ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs [hunk 2]
```diff
@@ -187,9 +187,21 @@ impl IndexerStreamCoordinator {
...
+                Ok(_) => (),
+                Err(e) => {
+                    // Error from closed channel. This means the client has disconnected.
+                    error!(
+                        start_version = first_version,
+                        end_version = end_version,
+                        num_transaction,
+                        highest_known_version,
+                        duration_in_secs = sending_start_time.elapsed().as_secs_f64(),
+                        error = format!("{:?}", e),
```
Comment (jillxuu) [inline ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs:200]: if potential error msgs could be verbose, we might wanna consider trim it down a bit , could set a pretty conservative number to begin with a

**Review by jillxuu — APPROVED** [review]

File: ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs (hunk 1 above)
Comment (rtso) [reply ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs:202]: fullnode latest version metric caught this error already

File: ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs (hunk 2 above)
Comment (rtso) [reply ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs:200]: i don't think we trim the error in other places in fullnode/grpc so it should be fine to leave as is. better to have the full error message than truncated version haha

**Pull request closed without merging**